File: src/main/java/br/com/correios/enderecador/bean/UsuarioBean.java
package br.com.correios.enderecador.bean;

public class UsuarioBean {
    private String pwd = "";

    private String usuarioProxy = "";

    private static UsuarioBean usuarioBean;

    public static UsuarioBean getInstance() {
        if (usuarioBean == null)
            usuarioBean = new UsuarioBean();
        return usuarioBean;
    }

    public String getPwd() {
        return this.pwd;
    }

    public void setUsuario(String usuarioProxy) {
        this.usuarioProxy = usuarioProxy;
    }

    public String getUsuario() {
        return this.usuarioProxy;
    }

    public void setPwd(String pwd) {
        this.pwd = pwd;
    }
}

File: src/main/kotlin/br/com/correios/enderecador/bean/UsuarioBean.kt
package br.com.correios.enderecador.bean

import org.koin.core.annotation.Singleton

@Singleton
class UsuarioBean {
    var pwd = ""
    var usuario = ""

    companion object {
        private var usuarioBean: UsuarioBean? = null
        @JvmStatic
        val instance: UsuarioBean?
            get() {
                if (usuarioBean == null) {
                    usuarioBean = UsuarioBean()
                }
                return usuarioBean
            }
    }
}
